Lib/typing.py

```diff
@@ -308,19 +308,44 @@ def _unpack_args(args):
             newargs.append(arg)
     return newargs
 
-def _deduplicate(params):
+def _deduplicate(params, *, unhashable_fallback=True):
```

Comment:
I'm worried that there's no way of avoiding abysmal performance of `_deduplicate` for unhashable elements. Could we make the default `False` for this parameter?